io.github.bothlab.syntalos.yaml

```
@@ -230,8 +229,8 @@ modules:
     - -DGSTREAMER_1.0_INCLUDEDIR=/include/gstreamer-1.0
   sources:
     - type: git
-      commit: 2225d34d2f6be6134965b504d56fc7388c1d67f7
       url: https://github.com/TheImagingSource/tiscamera.git
+      branch: v-tiscamera-1.0.0
```

no branch. a commit is necessary.

---

```suggestion
      tag: v-tiscamera-1.0.0
```

since it's actually a tag.